helmfile.d/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -4,6 +4,10 @@ pspEnabled: false
 
 config:
   modules:
+    http_2xx:
+      http:
+        tls_config:
+          insecure_skip_verify: {{ not .Values.global.verifyTls }}
```

Comment:
Should we (is it possible) add a comment here to discuss that it's not recommended, but might be required in some Air-gapped environments.